### ISSUE TITLE
porter.py returns an error in self._step1b(w) when w=vowel+'ing' (Python 3)

### ISSUE THREAD

**Issue by duartedenio** [nltk/nltk#1757]
Any word in the form vowel+'ing' makes PorterStemmer to abort.

```
stemmer = PorterStemmer()
word='oing'
stemmer.stem(word)
>>> File "/usr/local/lib/python3.5/dist-packages/nltk/stem/porter.py", line 214, in ends_double_consonant
>>>    word[-1] == word[-2] and 
>>> IndexError: string index out of range
```

The same error for aing, eing, iing, and uing. However, no errors if it is  consonant+'ing', e.g., ting, bing, ...

**Comment by alvations** [comment]
@duartedenio this issue has been resolved by @ExplodingCabbage previously #1582

Please upgrade your NLTK version and you should see the fix =)

```
>>> import nltk
>>> nltk.__version__
'3.2.4'

>>> from nltk.stem import PorterStemmer
>>> stemmer = PorterStemmer()
>>> word='oing'
>>> stemmer.stem(word)
u'o'
```

**Comment by alvations** [comment]
Resolved in #1582 

Closing resolved issues to prepare for v3.2.5.
